### PR TITLE
waste-exemptions move to shared reorganisation 

### PR DESCRIPTION
**Pull request by autotelik**
Minimum moves ensuring specs still pass

1)
Move decorators from we to we-shared
Move strong_parameters.rb and permitted_attributes from we to we-shared
Move concerns from we to we-shared + namespace

2) Move generic controllers and associated routes

3) Move helpers from we to we-shared + namespace and add to required controllers

### CODE REVIEW

File: app/assets/stylesheets/_application_ie.scss [hunk 1]
```diff
@@ -16,5 +16,5 @@ $is-ie: true;
 @import './related';
 @import './reset';
 @import './tables';
-@import './tabs';
+/* TODO: - how ?? @import 'waste_exemptions_shared/_tabs' */
```
Comment (rickclare) [inline app/assets/stylesheets/_application_ie.scss]: Is this still an unresolved TODO?   Looks like you managed in in application.scss below?

File: config/locales/errors/en.yml [hunk 2]
```diff
@@ -92,4 +92,4 @@ en:
     you_must_make_selection: You must make a selection from the list
   shared:
     errors:
-      you_have: You have
+      you_have: You hav
```
Comment (rickclare) [inline config/locales/errors/en.yml]: This looks like a typo  "You hav"  (missing a 'e')

File: app/assets/stylesheets/_application_ie.scss (hunk 1 above)
Comment (autotelik) [inline app/assets/stylesheets/_application_ie.scss]: drat. yes still untested. I don't have i.e on my virtual machines so need to get browser stack reset up or something to be able to check

File: config/locales/errors/en.yml (hunk 2 above)
Comment (autotelik) [inline config/locales/errors/en.yml]: well spotted. fixed

File: app/assets/stylesheets/_application_ie.scss (hunk 1 above)
Comment (rickclare) [inline app/assets/stylesheets/_application_ie.scss]: Suggest we raise this as a Trello or Jira(bug) and fix after merging.

Comment (autotelik) [inline app/assets/stylesheets/_application_ie.scss]: sounds good to me

**Pull request merged**